src/main/kotlin/com/sourcegraph/cody/auth/ui/EditCodebaseContextAction.kt

```diff
@@ -105,10 +104,6 @@ class EditCodebaseContextAction(val project: Project) : AbstractAction("Cody Con
             }
             .rowComment("Example: github.com/sourcegraph/cody")
         row { text("The URL will be used to retrieve the code context from the Cody API.") }
-        row {
-          text(
-              "If left empty, Cody will automatically infer the URL from your project's Git metadata.")
```

Comment:
I do not understand 🤔 isn't it true?

Reply:
Correct. I you open the Context Selection dialog you can't leave it empty. I've explained that in the commit message/PR description.

This statement is also not valid on main branch.

Reply:
ack! thanks for the explanation